### ISSUE TITLE
Report Templates

### ISSUE THREAD

**Issue by mr-raptor** [reportunit/reportunit#52]
Hi! Thanks for project, it's very helpful to visualize test results.
I saw that you store razor templates as strings in special classes. But I would to store templates in *.cshtml files. In this case we get razor syntax highlighting, so it's more simple to work with template. Yea, there are many errors in error window, because we use Model object on the view, but our views aren't parameterized. But project builds successfully.
So I created  *.cshtml file with template and added it to project resources. And then I can access it by next code:
`System.Text.Encoding.UTF8.GetString(Resources.Reports)` ("Reports" it's name of view)
I think it's better method of storing templates than now.

**Comment by anshooarora** [comment]
Yes sure, that would work.  I had many errors which is why I had to stringify the template.  Can you send in a pull request for this if possible?  I will merge.

**Comment by mr-raptor** [comment]
Hi again! I created pull request for this task https://github.com/reportunit/reportunit/pull/73
Checked on VS 2012 and VS 2015. In 2012 there is no any error, but in 2015 I get one error in console. Anyway, it doesn't matter - IntelliSense works good, project builds and works. 
If all is OK please merge.
